agreement-service.js

```diff
@@ -78,7 +71,3 @@ function factory($http, config) {
 
   return service;
 }
```

Comment:
For the record: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export